File: Extra/maior.c
#include <stdio.h>

void maior(int a, int b){
    if(a == b) printf("Iguais!\n");
    else if(a > b) printf("'A' eh o mairo\n");
    else printf("'B' eh o maior\n");
}

int main(){

    int a,b;
    printf("Digite dois numeros: \n");
    scanf("%d %d", &a, &b);

    maior(a,b);

    return 0;
}

File: Extra/maior.py
def maior(a, b):
    if a == b:
        print("Iguais!")
    elif a > b:
        print("'A' eh maior")
    else:
        print("'B' eh maior")

print("Digite dois numeros:")
a,b = map(int, input().split())

maior(a,b)